### PR TITLE
Configure authentication with tymon/jwt auth package

### PR DESCRIPTION
**Pull request by elguitarraverde**
Configure authentication with tymon/jwt auth package.

### CODE REVIEW

**Comment by elguitarraverde** [comment]
now the middleware to use is `'auth:api'`

image

File: app/Http/Controllers/Api/Auth/LoginController.php [hunk 1]
```diff
@@ -2,6 +2,80 @@
 
 namespace App\Http\Controllers\Api\Auth;
 
-class LoginController
+use App\Http\Controllers\Controller;
+
+class LoginController extends Controller
```
Comment (gnovaro) [inline app/Http/Controllers/Api/Auth/LoginController.php:7]: In my opinion is better do this more SOLID and split each action to a custom controller. But we can refactor later

**Review by gnovaro — APPROVED** [review]
Great job. Thank you!

File: app/Http/Controllers/Api/Auth/LoginController.php (hunk 1 above)
Comment (elguitarraverde) [reply app/Http/Controllers/Api/Auth/LoginController.php:7]: its true, you are right.

**Pull request merged**